.github/e2e.sh

```diff
@@ -8,9 +9,7 @@ ROOT_DIR=$(dirname $SCRIPTPATH)
 VERSION=$(cat .version)
 VERSION_SAFE="${VERSION//./}"
 PHP_VERSION=$(php -r "echo PHP_MAJOR_VERSION.PHP_MINOR_VERSION;")
-SITENAME="${VERSION_SAFE}-php${PHP_VERSION//./}-${VCS_REF}"
-
-terminus site:delete "${SITENAME}" --yes --quiet &> /dev/null
+SITENAME="${VERSION_SAFE}-php${PHP_VERSION//./}-t${TERMINUS_VERSION//./}-${VCS_REF}"
```

Comment:
```suggestion
SITENAME="ci-${VERSION_SAFE}-php${PHP_VERSION//./}-t${TERMINUS_VERSION//./}-${VCS_REF}"
```
make cleaning these up easier...